bluemira/equilibria/fem_fixed_boundary/utilities.py

```diff
@@ -459,7 +460,14 @@ def find_extremum(
         )
 
         optimiser.add_eq_constraints(f_constraint, tolerance=1e-10)
-        return optimiser.optimise(x0)
+        try:
+            x_star = optimiser.optimise(x0)
+        except Exception as e:
```

Comment:
Can we have a more specific exception? This could easily hide other problems

Reply:
Done but writing a test for this will be hard, it tends to occur when things go haywire in the fixed boundary, and that only happens after a lot of stuff takes place first. Too much effort for very little reward, I feel.

Reply:
yeah I agree, I think the codecov check can be ignored for merging